instances/anatomicalPlane/axial.jsonld

```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/anatomicalPlane/axial",
   "@type": "https://openminds.ebrains.eu/controlledTerms/AnatomicalPlane",
-  "definition": "An anatomical plane which divides the body into cranial and caudal (head and tail) parts.",
+  "definition": "An anatomical plane which divides the body or parts of the body into superior and inferior (dorsal and ventral, or top and bottom) parts.",
   "description": null,
   "interlexIdentifier": null,
   "knowledgeSpaceLink": null,
```

Comment:
Could we add synonym?

  "synonym": [
    "Transverse",
    "Horizontal"
  ]